smtp/smtp.c

```diff
@@ -549,6 +549,7 @@ static void handle_auth(enum state *state)
 		if(user_end == NULL)
 			REPLY("535 Authentication credentials invalid")
 		username_size = (size_t)(user_end - user_start);
+		if (username_size > sizeof username) REPLY("535 Username too long");
```

Comment:
The body of the if should be on the next line like all the other one line conditionals in the file